src/inmanta/module.py

```diff
@@ -611,6 +707,36 @@ def __init__(self, local_repo: "ModuleRepo", remote_repo: "ModuleRepo") -> None:
         self.local_repo: ModuleRepo = local_repo
         self.remote_repo: ModuleRepo = remote_repo
 
+    def log_pre_install_information(self, project: "Project", module_name: str) -> Dict[str, "Version"]:
+
+        LOGGER.debug("Installing module %s (v1).", module_name)
+
+        version_snapshot = env.PythonWorkingSet.get_packages_in_working_set(inmanta_modules_only=True)
```

Comment:
Indeed. I overlooked this because I thought a V2 module might get installed as a dependency of the V1 modules. But this call only installs the V1 module itself and not its dependencies.

Reply:
Removed for v1 modules

Reply:
Do we have consistent logging for V1 and V2 then? 

I think the testcases should also include some V1's to verify consistency.